Makefile.include

```diff
@@ -141,6 +141,12 @@ ifeq ($(DEVELHELP),1)
   CFLAGS += -DDEVELHELP
 endif
 
+# If not already set in CFLAGS, set logging level by default to INFO, e.g. 3.
+LOG_LEVEL ?= 3
```

Comment:
Why not leave it unset and check if it is set as default? This way in case we want to change the default we don't have to do it in three places (here, the doc in `vars.inc.mk`, and the define) and confusions for a person trying to change just [the define](https://github.com/RIOT-OS/RIOT/blob/1dc479de00ed81981d7495e6e3c53b0a791b3efb/core/include/log.h#L70) are avoided.

Reply:
Ok, I changed it this way.